 <launch>
   <node pkg="baxter_tools" type="enable_robot.py" name="enable" args="-e" output="screen"/>
  <node pkg="baxter_trajectory_streamer" type="rt_arm_as" name="rt_arm_as"  output="screen"/>
  <node pkg="baxter_trajectory_streamer" type="left_arm_as" name="left_arm_as"  output="screen"/> 
  <node pkg="cartesian_planner" type="baxter_cart_move_as" name="baxter_cart_move_as" output="screen"/>
  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find coordinator)/rviz/coordinator_rviz.rviz"/> 
  <node pkg="baxter_playfile_nodes" type="baxter_playback" name="baxter_playback" args="$(find baxter_playfile_nodes)/pre_pose_right.jsp $(find baxter_playfile_nodes)/pre_pose_left.jsp" output="screen"/> 
  <node pkg="baxter_playfile_nodes" type="baxter_playfile_service" name="baxter_playfile_service"  output="screen"/>  
  <node pkg="example_rviz_marker" type="triad_display" name="triad_display"/>
  <node pkg="object_grabber" type="object_grabber_as" name="object_grabber_as"  output="screen"/>
  <node pkg="object_finder" type="object_finder_as" name="object_finder_as"  output="screen"/>  
  <node pkg="coordinator" type="coordinator_v4" name="coordinator_v4"  output="screen"/>
</launch>
